notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -416,26 +385,26 @@
+        "4. Upload schema or use automatic generation of the *input schema* for parsing.\n",
+        "5. For feature skewness detection, upload the training data. This enables automatic generation of the feature distributions.\n",
+        "6. For feature attributions, upload corresponding *Vertex Explainable AI* specification.\n",
         "\n",
-        "Once configured, you can enable/disable monitoring, change alerts and update the model monitoring configuration. \n",
+        "Once configured, you can enable or disable monitoring, change alerts and update the model monitoring configuration. \n",
         "\n",
-        "When model monitoring is enabled, the sampled incoming prediction requests are logged into a BigQuery table. The input feature values contained in the logged requests are then analyzed for skew or drift on an specified interval basis. You set a sampling rate to monitor a subset of the production inputs to a model, and the monitoring interval.\n",
+        "When model monitoring is enabled, the sampled incoming prediction requests are logged into a BigQuery table. The input feature values contained in the logged requests are then analyzed for skewness or drift on an specified interval basis. You set a sampling rate to monitor a subset of the production inputs to the model, and the monitoring interval.\n",
         "\n",
         "The model monitoring service needs to know how to parse the feature values, which is referred to as the input schema. For AutoML tabular models, the input schema is automatically generated. For custom tabular models, the service attempts to automatically derive the input schema from the first 1000 prediction requests. Alternatively, one can upload the input schema.\n",
         "\n",
-        "For skew detection, the monitoring service requires a baseline for the statistical distribution of values in the training data. For AutoML tabular models this is automatically derived. For custom tabular models, you upload the training data to the service, and have the service automatically derive the distribution.\n",
+        "For skewness detection, the monitoring service requires a baseline for the statistical distribution of values in the training data. For AutoML tabular models this is automatically derived. For custom tabular models, you upload the training data to the service, and have the service automatically derive the distribution.\n",
         "\n",
-        "For feature attribution skew and drift detection, requires enabling your deployed model for `Vertex AI Explainability` for custom tabular models. For AutoML models, `Vertex AI Explainability` is automatically enabled.\n",
+        "For feature attribution skewness and drift detection in feature attributions, you're required to enable Vertex Explainable AI feature for your deployed custom tabular models. For AutoML models, Vertex Explainable AI is automatically enabled.\n",
```

Comment:
change (missing "the"):

-  you're required to enable Vertex Explainable AI feature 

TO:

-  you're required to enable the Vertex Explainable AI feature